nym-wallet/src-tauri/tauri.conf.json

```diff
@@ -23,10 +23,10 @@
       ],
       "resources": [],
       "externalBin": [],
-      "copyright": "",
-      "category": "DeveloperTool",
-      "shortDescription": "",
-      "longDescription": "",
+      "copyright": "Copyright © 2021-2022 Nym Technologies SA",
+      "category": "Business",
+      "shortDescription": "Nym desktop wallet allows you to manage your Nym tokens",
+      "longDescription": "The Nym wallet allows you to manage your Nym tokens and to stake on mixnodes, gateways and validators.",
```

Comment:
Well, technically you can't stake on gateways (at all) and the wallet itself doesn't let you to stake on validators